src/events/pg.js

```diff
@@ -1,8 +1,8 @@
 const uuid4 = require('uuid4');
 const shimmer = require('shimmer');
-const pg = require('pg');
-const Pool = require('pg-pool');
-const sqlParser = require('node-sqlparser');
+const pg = require('pg'); // eslint-disable-line import/no-unresolved
```

Comment:
don't you have to use try-require here or something? or is it somewhere else?

Reply:
The try-require is being used in patcher.js (try to require the whole `events/pg.js`)